src/main/java/com/arpnetworking/metrics/mad/Main.java

```diff
@@ -205,6 +206,9 @@ private void launchActors(final Injector injector) {
         // Create the telemetry connection actor
         actorSystem.actorOf(Props.create(Telemetry.class, injector.getInstance(MetricsFactory.class)), "telemetry");
 
+        // Create the actor sources supervisor actor
+        actorSystem.actorOf(SourceSupervisor.props(), "source");
```

Comment:
There was confusion around what was being asked.  The question was actually something like "Why isn't the source class an actor?"  The answer is that the Source class has a prescribed threading model with it's implementation of Launchable as well as a specific life cycle (for the same reason).  The actor lives in a different threading model and needs to be created by the ActorSystem.  The actual Actor being an internal class of the Source allows for this decoupling.